lib/ex_doc/formatter/html/templates.ex

```diff
@@ -30,6 +30,27 @@ defmodule ExDoc.Formatter.HTML.Templates do
     nil
   end
 
+  @doc """
+  Get the attribute string used to define the spec of the given `node`.
+  """
+  def get_spec_attribute(module, node) do
+    node
+    |> get_spec_attribute()
+    |> module.language.format_attribute()
+  end
+
+  def get_spec_attribute(%ExDoc.TypeNode{} = node) do
+    if "opaque" in node.annotations do
+      "opaque"
+    else
+      "type"
+    end
```

Comment:
I believe this should work, wdyt?

```suggestion
    Atom.to_string(node.type)
```

Reply:
Huh, funny, I could have sworn I got `type: :type` previously. Yes this works!